ibm/service/vpc/resource_ibm_is_backup_policy.go

```diff
@@ -0,0 +1,407 @@
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting resource_group: %s", err))
+		}
+	}
+	if backupPolicy.CreatedAt != nil {
+		if err = d.Set("created_at", flex.DateTimeToString(backupPolicy.CreatedAt)); err != nil {
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting created_at: %s", err))
+		}
+	}
+
+	if backupPolicy.CRN != nil {
+		if err = d.Set("crn", backupPolicy.CRN); err != nil {
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting crn: %s", err))
+		}
+	}
+
+	if backupPolicy.Href != nil {
+		if err = d.Set("href", backupPolicy.Href); err != nil {
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting href: %s", err))
+		}
+	}
+
+	if backupPolicy.LastJobCompletedAt != nil {
+		if err = d.Set("last_job_completed_at", flex.DateTimeToString(backupPolicy.LastJobCompletedAt)); err != nil {
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting last_job_completed_at: %s", err))
+		}
+	}
+
+	if backupPolicy.LifecycleState != nil {
+		if err = d.Set("lifecycle_state", backupPolicy.LifecycleState); err != nil {
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting lifecycle_state: %s", err))
+		}
+	}
+
+	if backupPolicy.ResourceType != nil {
+		if err = d.Set("resource_type", backupPolicy.ResourceType); err != nil {
+			return diag.FromErr(fmt.Errorf("[ERROR] Error setting resource_type: %s", err))
+		}
+	}
+
+	if err = d.Set("version", response.Headers.Get("Etag")); err != nil {
+		return diag.FromErr(fmt.Errorf("[ERROR] Error setting version: %s", err))
+	}
+
+	return nil
+}
+func resourceIBMIsBackupPolicyResourceGroupIdentityToMap(resourceGroupIdentity vpcv1.ResourceGroupReference) map[string]interface{} {
+	resourceGroupIdentityMap := map[string]interface{}{}
+	resourceGroupIdentityMap["id"] = resourceGroupIdentity.ID
+	return resourceGroupIdentityMap
+}
+
+func resourceIBMIsBackupPolicyUpdate(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	vpcClient, err := vpcClient(meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	updateBackupPolicyOptions := &vpcv1.UpdateBackupPolicyOptions{}
+	updateBackupPolicyOptions.SetID(d.Id())
+	hasChange := false
+	patchVals := &vpcv1.BackupPolicyPatch{}
+	if d.HasChange("match_user_tags") {
+		patchVals.MatchUserTags = (flex.ExpandStringList((d.Get("match_user_tags").(*schema.Set)).List()))
+		hasChange = true
+	}
+	if d.HasChange("name") {
+		patchVals.Name = core.StringPtr(d.Get("name").(string))
+		hasChange = true
+	}
+	updateBackupPolicyOptions.SetIfMatch(d.Get("version").(string))
+	if hasChange {
+		updateBackupPolicyOptions.BackupPolicyPatch, _ = patchVals.AsPatch()
+		_, response, err := vpcClient.UpdateBackupPolicyWithContext(context, updateBackupPolicyOptions)
+		if err != nil {
+			log.Printf("[DEBUG] UpdateBackupPolicyWithContext failed %s\n%s", err, response)
+			return diag.FromErr(fmt.Errorf("[ERROR] UpdateBackupPolicyWithContext failed %s\n%s", err, response))
+		}
+	}
+
+	if d.HasChange("plans") && !d.IsNewResource() {
```

Comment:
we don't have "plans" defined in the schema

Reply:
revised to correct ones.